plugin/src/main/kotlin/nl/avisi/anowalke/service/RestService.kt

```diff
@@ -22,21 +22,21 @@ class RestService {
     var client: OkHttpClient = OkHttpClient()
 
     val gson = Gson()
-    val JSON: MediaType? = ("application/json; charset=utf-8").toMediaTypeOrNull()
+    val json: String = "application/json; charset=utf-8"
+    val mediaType: MediaType = json.toMediaType()
 
 
     @Throws(IOException::class)
     fun makeApiCall(expressions: ExpressionsDto, url: String): List<String> {
         val requestBody = gson.toJson(expressions)
-        log.warn(requestBody)
-        val body: RequestBody = requestBody.toRequestBody(JSON)
+        log.info(requestBody)
+        val body: RequestBody = requestBody.toRequestBody(mediaType)
         val request: Request = Request.Builder()
             .url(url).post(body)
-            .addHeader("Content-type", "application/json").build()
+            .addHeader("Content-type", json).build()
         val response: Response = client.newCall(request).execute()
-        log.warn("Response received with status: ${response.code}")
-        val responseList = gson.fromJson(response.body?.string(), List::class.java)
-        return responseList.map { it.toString() }
+        log.info("Response received with status: ${response.code}")
+        return gson.fromJson(response.body?.string(), List::class.java).map { it.toString() }
```

Comment:
Met direct omzetten bedoelde ik niet dat het direct achter de fromJson call komt. Ik doelde erop dat je als type niet List::class.java meegeeft, maar direct het type List<String> Dan zou dat direct een lijst van strings kunnen zien in plaats van dat je de hele lijst maakt en die dan nog een keer moet omzetten.

Reply:
Voor zover ik me kan herinneren pakt hij dat niet.. ik kan niet meegeven dat het List<String> moet worden..